beacon-chain/rpc/beaconv1/server_test.go

```diff
@@ -2,4 +2,4 @@ package beaconv1
 
 import ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1"
 
-var _ ethpb.BeaconChainServer = (*Server)(nil)
+var _ = ethpb.BeaconChainServer(&Server{})
```

Comment:
Revert this. `var _ ethpb.BeaconChainServer = (*Server)(nil)` is preferred way to check interface conformance.